src/feature/relay/dns.c

```diff
@@ -2026,6 +2033,26 @@ dns_launch_correctness_checks(void)
   }
 }
 
+/** If appropriate, if IPv6 DNS failed, reset the countner every 24 hours */
+static void
+dns_launch_ipv6_broken_reset(void)
+{
+  static struct event *launch_event = NULL;
+  struct timeval timeout;
+
+  if (dns_was_broken_for_ipv6)
+    return;
+
+  if (!launch_event)
+    launch_event = tor_evtimer_new(tor_libevent_get_base(),
+                                   dns_reset_ipv6_broken, NULL);
+  timeout.tv_sec = 24 * 60 * 60;
```

Comment:
Please define this value using a named macro, rather than just doing the calculation directly in the code.

Named constants help people understand the code.